Marlin/src/feature/caselight.cpp

```diff
@@ -69,7 +69,7 @@ void update_case_light() {
 
   #else // !CASE_LIGHT_USE_NEOPIXEL
 
-    if (PWM_PIN(CASE_LIGHT_PIN))
+    if (PWM_PIN(CASE_LIGHT_PIN) && DISABLED(CASE_LIGHT_NO_BRIGHTNESS))
```

Comment:
You can't use `DISABLED` like this. It won't expand. You have to turn it into a run-time-accessible boolean.

Reply:
I blame lack of coffee.... I dont know what I was thinking on that one lol